spec/punctuation/hyphen_spec.rb

```diff
@@ -221,9 +221,9 @@
   end
 
   context "'customer-driven academic library'", :jira => ['SW-388', 'VUF-846'] do
-    it_behaves_like "hyphens without spaces imply phrase", "customer-driven academic library", "7778647", 1
-    it_behaves_like "hyphens ignored", "customer- driven academic library", "7778647", 1
-    it_behaves_like "hyphens with space before but not after are treated as NOT, but ignored in phrase", "customer -driven academic library", nil, "7778647"
+    it_behaves_like "hyphens without spaces imply phrase", "customer-driven academic library", "12582003", 1
```

Comment:
I don’t think we should change the ckey that is getting checked. The new ckey is an eloader record, which has a higher probability of being replaced through some Tech Services process than the ckey for the print copy in Sal3.

Reply:
ok, we can revert that and just look a couple records more?